### PR TITLE
Conflict free branches of Fulgur14

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

**Review by Fulgur14 — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/the_capitoline_triad.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:The Capitoline Triad
+ManaCost:10
+Types:Legendary Creature God Artificer
+PT:7/7
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Those Who Came Before — This spell costs {1} less to cast for each historic card in your graveyard. (Artifacts, legendaries, and Sagas are historic.)
+A:AB$ Effect | Cost$ ExileFromGrave<1+/Card.Historic+withTotalCMCEQ30/historic cards from your graveyard with total mana value 30 or greater> | Name$ Emblem - The Capitoline Triad | Image$ emblem_the_capitoline_triad | StaticAbilities$ STSetPT | Stackable$ False | Duration$ Permanent | AILogic$ Always | SpellDescription$ You get an emblem with "Creatures you control have base power and toughness 9/9."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_capitoline_triad.txt:6]: this still needs engine support though

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/the_capitoline_triad.txt:6]: And that should probably wait until we're SURE it's real :)

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/the_capitoline_triad.txt:6]: @Simisays can we skip this file for a later Commit?

File: forge-gui/res/cardsfolder/k/khod_etlan_shiis_envoy.txt [hunk 2]
```diff
@@ -3,5 +3,5 @@ ManaCost:2 U
 Types:Legendary Creature Homarid Warrior
 PT:2/3
 S:Mode$ Continuous | Affected$ Land | AddType$ Island | Description$ All lands are Islands in addition to their other types.
-S:Mode$ Continuous | Affected$ Homarid.Other+YouCtrl,Camarid.Other+YouCtrl,Cephalid.Other+YouCtrl,Nautilid.Other+YouCtrl,Merfolk.Other+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Other Homarids, Camarids, Cephalids, Nautilids, and Merfolk you control get +1/+1.
-Oracle:All lands are Islands in addition to their other types.\nOther Homarids, Camarids, Cephalids, Nautilids, and Merfolk you control get +1/+1.
+S:Mode$ Continuous | Affected$ Homarid.Other+YouCtrl,Camarid.Other+YouCtrl,Octopus.Other+YouCtrl,Nautilid.Other+YouCtrl,Merfolk.Other+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Other Homarids, Camarids, Cephalids, Nautilids, and Merfolk you control get +1/+1.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/k/khod_etlan_shiis_envoy.txt:6]: forgot one `Cephalids` there to change to `Octopuses`

**Comment by tool4ever** [comment]
TypeLists should be updated too

**Review by Hanmac — APPROVED** [review]

**Pull request merged**